app/src/pages/inside/testPlansPage/testPlansTable/testPlansTable.scss

```
@@ -112,8 +112,29 @@ $test-plans-progress-column-inline-pad: 16px;
+    font-family: $FONT-ROBOTO-MEDIUM;
+    font-size: 13px;
+    font-weight: 500;
+    line-height: 20px;
+    padding-left: 1px;
+    letter-spacing: 0;
+    color: var(--Text-Secondary, var(--rp-ui-base-e-400));
+  }
+
   &__plan-name {
     @include ms.milestone-test-plans-cell-name;
+    flex: 1;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)